website/docs/docs/use-dbt-semantic-layer/dbt-sl.md

```diff
@@ -0,0 +1,269 @@
+import NewSLChanges from '/snippets/_new-sl-changes.md';
+
+<NewSLChanges />
+
+
+The dbt Semantic Layer, powered by MetricFlow, simplifies the process of defining and using critical business metrics, like like `revenue` in the modeling layer (your dbt project). By centralizing metric definitions, data teams can ensure consistent self-service access to these metrics in downstream data tools and applications. The dbt Semantic Layer eliminates duplicate coding by allowing you to define metrics on top of existing models and automatically handles data joins. 
+
+By leveraging the dbt Semantic Layer, data teams achieve reduced code duplication and increased consistency in business metrics across different applications. Moving metric definitions out of the BI layer and into the modeling layer allows data teams to feel confident that different business units are working from the same metric definitions, regardless of their tool of choice. If a metric definition changes in dbt, it’s refreshed everywhere it’s invoked and creates consistency across all applications. 
+
+To read more about why you need a universal Semantic Layer, read this [blog post](https://www.getdbt.com/blog/universal-semantic-layer/).
+
+The dbt Semantic Layer has the following major components and you can refer to [product architecture](#product-architecture) for more details:
+
+- [MetricFlow](/docs/build/build-metrics-intro) allows you to define and test your semantic models and metrics in version-controlled dbt project code.
+- [dbt Semantic Layer APIs](/docs/use-dbt-semantic-layer/sl-api-overview) enables you to explore and analyze these metrics in downstream tools.
+
+## Explore the dbt Semantic Layer
+
+The dbt Semantic Layer is available to all dbt Cloud plans (Developer, Team, and Enterprise) during [public beta](#public-beta). Start exploring the dbt Semantic Layer by visiting the following pages:
+
+<div className="grid--2-col">
+
+<Card
+    title="Get started with MetricFlow and the dbt Semantic Layer"
+    body="Build and define metrics, set up the dbt Semantic Layer, and query them using the Semantic Layer APIs."
+    link="/docs/use-dbt-semantic-layer/quickstart-sl"
+    icon="dbt-bit"/>
+
+<Card
+    title="Set up the dbt Semantic Layer"
+    body="Set up the dbt Semantic Layer in dbt Cloud using intuitive navigation."
+    link="/docs/use-dbt-semantic-layer/setup-sl"
+    icon="dbt-bit"/>
+
+<Card
+    title="Available integrations"
+    body="Review a wide range of partners you can integrate and query with the dbt Semantic Layer."
+    link="/docs/use-dbt-semantic-layer/avail-sl-integrations"
+    icon="dbt-bit"/>
+    
+<Card
+    title="Semantic Layer API"
+    body="Use the Semantic Layer APIs to query metrics in downstream tools."
+    link="/docs/use-dbt-semantic-layer/sl-api-overview"
+    icon="dbt-bit"/>
+
+</div>
+
+
+## Public beta 
+
+MOVING THIS TO RELEASE NOTE SECTION BECAUSE I DONT THINK THIS BELONGS HERE. ANYTHING THAT SAYS PUBLIC BETA CAN LINK TO RN
```

Comment:
i think public beta should not be on this pag and be announced/detailed in the public beta release note. any reference to it can then link to that page. this way, it keeps this page clean and only maintains the relevant/current info